components/infobox/wikis/starcraft2/infobox_unit_custom.lua

```diff
@@ -24,20 +25,24 @@ local CustomUnit = Class.new()
 
 local CustomInjector = Class.new(Injector)
 
-local _ICON_MINERALS = '[[File:Minerals.gif|baseline|link=Minerals]]'
-local _ICON_GAS = mw.loadData('Module:Gas')
-local _ICON_TIME = mw.loadData('Module:Buildtime')
-local _ICON_SUPPLY = mw.loadData('Module:Supply')
-local _ICON_HP = '[[File:Icon_Hitpoints.png|link=]]'
-local _ICON_SHIELDS = '[[File:Icon_Shields.png|link=Plasma Shield]]'
-local _ICON_ARMOR = '[[File:Icon_Armor.png|link=Armor]]'
+local ICON_MINERALS = '[[File:Minerals.gif|baseline|link=Minerals]]'
+local ICON_GAS = mw.loadData('Module:Gas')
+local ICON_TIME = mw.loadData('Module:Buildtime')
+local ICON_SUPPLY = mw.loadData('Module:Supply')
+local ICON_HP = '[[File:Icon_Hitpoints.png|link=]]'
+local ICON_SHIELDS = '[[File:Icon_Shields.png|link=Plasma Shield]]'
+local ICON_ARMOR = '[[File:Icon_Armor.png|link=Armor]]'
+local LOTV = 'lotv'
```

Comment:
```suggestion
local LOTV = Game.name{game = 'lotv'}
```

Same in building infobox (the others already use this pattern). Remember to update the if statement as well.